### PR TITLE
RDoc-2243 Add example for Storage.OnDirectoryInitialize.Exec + organi…

### PR DESCRIPTION
**Pull request by Danielle9897**
…ze directory-structure + fixed relevant configuration options

### CODE REVIEW

**Comment by Danielle9897** [comment]
@arekpalinski @gregolsky 
* Can you pls review files:
` customizing-raven-data-files-locations.markdown` + `directory-structure.markdown`

* Note: The script in the newly added example is still not tested yet, as I am waiting for my local Ubunto installation to complete

File: Documentation/4.1/Raven.Documentation.Pages/server/storage/customizing-raven-data-files-locations.markdown [hunk 1]
```diff
@@ -1,148 +1,228 @@
...
+  By default, all databases' temporary files are written to the `Temp` folder under each Database directory.  
+  Customize the files path by setting configuration option [Storage.TempPath](../..//server/configuration/storage-configuration#storage.temppath) in your _settings.json_ file.  
+
+* **Indexes temporary files**  
+  By default, all indexes' temporary files are written to the `Temp` folder under each Index directory.  
+  Customize the files path by setting configuration option [Indexing.TempPath](../../server/configuration/indexing-configuration#indexing.temppath) in your _settings.json_ file.  
+
+* **Backup temporary files**  
+  By default, backup temporary files are written under the `Database` directory or under `Storage.TempPath` if defined.  
+  Customize the files path by setting configuration option [Backup.TempPath](../../server/configuration/todo..) in your _settings.json_ file.
```
Comment (arekpalinski) [inline Documentation/4.1/Raven.Documentation.Pages/server/storage/customizing-raven-data-files-locations.markdown:47]: I think you already added it below so you can remove TODO here.

Comment (Danielle9897) [reply Documentation/4.1/Raven.Documentation.Pages/server/storage/customizing-raven-data-files-locations.markdown:47]: done

**Review by arekpalinski — APPROVED** [review]

**Comment by Danielle9897** [comment]
@ppekrol - pls don't merge this yet - I am applying @gregolsky comments

File: Documentation/4.1/Raven.Documentation.Pages/server/storage/customizing-raven-data-files-locations.markdown [hunk 2]
```diff
@@ -1,148 +1,236 @@
...
+
+After the script is run, the Indexes link will reside under the Database directory (See [Directory Structure](../../server/storage/directory-structure)).  
+The link will point to the new location set by variable $INDEXES_TARGET_DIR_NAME where all indexes' data will be written.  
+
+{CODE-BLOCK:bash}
+#!/bin/bash
+# bash ./your-script USER_ARGS Database DB_NAME BASE_PATH TEMP_PATH JOURNALS_PATH
+
+RDB_DATA_DIR="$RAVENDB_DATA_DIR"
+INDEXES_TARGET_DIR="$RAVENDB_INDEXES_TARGET_DIR"
```
Comment (gregolsky) [inline Documentation/4.1/Raven.Documentation.Pages/server/storage/customizing-raven-data-files-locations.markdown:182]: Maybe we need to add a comment or 

```suggestion
# NOTE: Use directory names you as defined on your machine
RDB_DATA_DIR="/var/lib/ravendb/data"
INDEXES_TARGET_DIR="/mnt/ravendb-indexes"
```

Comment (Danielle9897) [reply Documentation/4.1/Raven.Documentation.Pages/server/storage/customizing-raven-data-files-locations.markdown:182]: ok ?

```
# Use directory names as defined on your machine
RDB_DATA_DIR="/Server/RavenData"
INDEXES_TARGET_DIR="/mnt/ravendb-indexes"
```

Comment (gregolsky) [reply Documentation/4.1/Raven.Documentation.Pages/server/storage/customizing-raven-data-files-locations.markdown]: `/var/lib/ravendb/data` is a directory Ubuntu DEB package would use to store RavenDB data by default. `/Server/RavenData` looks artificial and very linux-savvy

**Review by gregolsky — APPROVED** [review]

**Comment by Danielle9897** [comment]
@ppekrol
 pr can be merged

**Pull request merged**